lib/config.js

```diff
@@ -49,6 +49,8 @@ const Config = function () {
   this.sccache = getNPMConfig(['sccache'])
   this.braveReferralsApiKey = getNPMConfig(['brave_referrals_api_key']) || ''
   this.ignore_compile_failure = false
+  this.enable_hangout_services_extension = true
```

Comment:
we should make sure that anything new we add here is android compatible. I'm guessing this needs to be !android

Reply:
Actually this can't be set yet because we don't have the android option landed yet. It'll have to go in in the branch where this is being added.